.github/dependabot.yml

```diff
@@ -1,3 +1,4 @@
+# yaml-language-server: $schema=https://json.schemastore.org/dependabot-2.0.json
```

Comment:
This seems redundant. The schemastore plugin in NeoViM gives schema hints with and without this line.